### DISCUSSION TITLE
When I start my zsh there is another zsh process that takes 99% of cpu (2 in total).

### DISCUSSION THREAD

**Discussion by aavidan** [ohmyzsh/ohmyzsh#12549] [Help]
top command on MacOs.
<img width="230" alt="image" src="https://github.com/ohmyzsh/ohmyzsh/assets/173013105/6d0c967c-7d43-42f0-a9a3-015365153ecc">
It takes high cpu and when I kill my terminal it stays in the activity monitor.
Sometimes I get multiple zsh processes dozens of them.
I would like to know if someone had this issue.
Thanks

**Comment by mcornella** [comment]
Please raise a bug report via [this form](https://github.com/ohmyzsh/ohmyzsh/issues/new?assignees=&labels=&projects=&template=bug_report.yml).
